assets/create.sh

```diff
@@ -3,6 +3,10 @@ set -euo pipefail
 
 ASSET_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+# Ensure submodules are initialized.
+git submodule init
```

Comment:
To make this a little bit more robust, you could use `git -C "$ASSET_DIR" …`. Otherwise, it only works when the CWD is within the `bat` repository (and not within one of the submodules).